controllers/keda/scaledobject_controller.go

```diff
@@ -246,6 +246,11 @@ func (r *ScaledObjectReconciler) reconcileScaledObject(ctx context.Context, logg
 		return "ScaledObject doesn't have correct Idle/Min/Max Replica Counts specification", err
 	}
 
+	err = r.checkTriggerNamesAreUnique(scaledObject)
```

Comment:
Not sure, whether we should check this. @tomkerkhove @JorTurFer

Reply:
IDK where either, there seems a good place xD

Reply:
I think it's good to validate uniquness but not 100% up to speed on this feature

Reply:
Let's go ahead with this then :)